target/linux/x86/image/Makefile

```diff
@@ -40,13 +38,17 @@ BOOTOPTS:=$(call qstrip,$(CONFIG_GRUB_BOOTOPTS))
 
 define Build/combined
 	$(CP) $(KDIR)/$(KERNEL_NAME) $@.boot/boot/vmlinuz
-	-$(CP) $(STAGING_DIR_ROOT)/boot/. $@.boot/boot/
```

Comment:
See? This is where the below L48 comes from.

Reply:
I'm wondering under what circumstances would this command fail?